plugins/wasp/README.md

```diff
@@ -0,0 +1,98 @@
+# Wasp Claude Code Plugin
+
+A Claude Code plugin for Wasp framework development, providing optimized raw text Wasp documentation for LLMs, best practices, and tested workflows.
+
```

Comment:
Let's amke this one match whatever we decide to go with in the other places?

Some things I don't like in this one specifically:
- "wasp framework development" -> that sounds like it is used for developing the wasp itself, not using it for development
- "raw text Wasp documentation" -> confusing, too low level